R/02-filter.R

```diff
@@ -75,14 +82,51 @@ gimap_filter <- function(.data = NULL,
   #then it finds the row sum (how many are filters flagged each construct e.g., number of TRUE in each row), 
   #and finally compares the row sum to the `min_n_filters` parameter to report TRUEs and FALSEs according to whether each construct is flagged by the minimum number of required filters
   #TRUE means it should be filtered, FALSE means it shouldn't be filtered
-  combined_filter <- rowSums(reduce(possible_filters, cbind)) >= min_n_filters
+  one_filter_df <- reduce(possible_filters, cbind) %>% 
+    `colnames<-`(c("FilterZeroCount", "FilterLowPlasmidCPM")) #*ADD any new filter's name here* as an additional column name; START with "Filter"
```

Comment:
```suggestion
    `colnames<-`(c("filter_zero_count", "filter_low_plasmi_cpm")) #*ADD any new filter's name here* as an additional column name; START with "Filter"
```

This is probably nitpicky of me, but elsewhere we've been using underscores instead of camel case so do you mind if we stick with that for consistency? (How much is this could to mess up other parts of your code? I can run a search and replace)

Reply:
a starts_with on line 103 will need edited I think